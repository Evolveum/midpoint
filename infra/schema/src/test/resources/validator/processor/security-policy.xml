<!--
  ~ Copyright (C) 2010-2023 Evolveum and contributors
  ~
  ~ This work is dual-licensed under the Apache License 2.0
  ~ and European Union Public License. See LICENSE file for details.
  -->

<securityPolicy xmlns="http://midpoint.evolveum.com/xml/ns/public/common/common-3"
        oid="28676177-5884-4625-805d-bae552932666">

    <name>security policy</name>

    <authentication>
        <modules>
            <saml2 id="1">
                <name>deprecated items</name>
                <network>
                    <connectTimeout>1</connectTimeout>
                    <readTimeout>2</readTimeout>
                </network>
            </saml2>
        </modules>
        <sequence id="2">
            <name>sequencename</name>
            <module id="3">
                <name>deprecated items</name>
            </module>
            <module id="4">
                <identifier>unchanged</identifier>
            </module>
        </sequence>
    </authentication>
    <credentialsReset>
        <name>sample-name</name>
    </credentialsReset>
</securityPolicy>
