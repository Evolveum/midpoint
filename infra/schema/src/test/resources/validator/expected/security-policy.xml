<!--
  ~ Copyright (C) 2010-2023 Evolveum and contributors
  ~
  ~ This work is dual-licensed under the Apache License 2.0
  ~ and European Union Public License. See LICENSE file for details.
  -->

<securityPolicy xmlns="http://midpoint.evolveum.com/xml/ns/public/common/common-3"
        oid="28676177-5884-4625-805d-bae552932666">

    <name>security policy</name>

    <authentication>
        <modules>
            <saml2 id="1">
                <identifier>deprecated items</identifier>
            </saml2>
        </modules>
        <sequence id="2">
            <identifier>sequencename</identifier>
            <module id="3">
                <identifier>deprecated items</identifier>
            </module>
            <module id="4">
                <identifier>unchanged</identifier>
            </module>
        </sequence>
    </authentication>
    <credentialsReset>
        <identifier>sample-name</identifier>
    </credentialsReset>
</securityPolicy>
